wiki/Disambiguation/id.md

```diff
@@ -1,10 +1,10 @@
 # Disambiguasi
 
-Artikel **disambiguasi** di [wiki osu!](/wiki/Main_Page) menyoroti istilah-istilah yang ambigu dan menjelaskan arti yang memungkinkan dari istilah tersebut.
+Dalam [osu! wiki](/wiki/Main_Page), artikel **disambiguasi** merupakan artikel yang membahas seputar kata-kata dan istilah-istilah yang dapat memiliki lebih dari satu arti beserta penjelasan singkat untuk masing-masing pengertiannya.
```

Comment:
```suggestion
Dalam [osu! wiki](/wiki/Main_Page), artikel **disambiguasi** merupakan artikel yang membahas seputar istilah-istilah, yang ada pada permainan osu!,yang dapat memiliki lebih dari satu arti beserta penjelasan singkat untuk masing-masing pengertiannya.
```

Reply:
o i applied something else based on this suggestion, tyty